src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrManagerBase.cs

```diff
@@ -177,7 +177,7 @@ protected async Task ReapplyVmrPatchesAsync(
             cancellationToken.ThrowIfCancellationRequested();
         }
 
-        await CommitAsync("[VMR patches] Re-apply VMR patches");
+        await CommitAmendAsync();
```

Comment:
This one is a bit weird in the non-code-flow flows (so for the regular sync) because there this creates a work branch and keeps making commits 1 per repo + these special ones like patch reapplication. This means that there will be commit like this:
- Remove patches
- Patches for sdk
- Patches for runtime
- Patches for ...
- Reapply patches

And then this would basically clump the reapply with the last repo.

Now this might be a problem or not really because we then squash merge the work branch anyway but might get confusing when debugging.

Reply:
we could pass a flag from the `CodeFlower` to `UpdateRepository`, and then to this method

Reply:
added a flag to make sure the regular sync keeps working as usual